sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -2162,6 +2296,65 @@ public partial class EventDataInfo : System.ClientModel.Primitives.IJsonModel<Az
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.EventDataInfo>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.EventDataInfo>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    public partial class Exporter : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.Exporter>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Exporter>
+    {
+        public Exporter(Azure.ResourceManager.Monitor.Models.ExporterType exporterType, string name) { }
+        public Azure.ResourceManager.Monitor.Models.AzureMonitorWorkspaceLogsExporter AzureMonitorWorkspaceLogs { get { throw null; } set { } }
+        public Azure.ResourceManager.Monitor.Models.ExporterType ExporterType { get { throw null; } set { } }
+        public string Name { get { throw null; } set { } }
+        public System.Uri TcpUri { get { throw null; } set { } }
+        Azure.ResourceManager.Monitor.Models.Exporter System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.Exporter>.Create(ref System.Text.Json.Utf8JsonReader reader, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.Exporter>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.Monitor.Models.Exporter System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Exporter>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Exporter>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Exporter>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ExporterType : System.IEquatable<Azure.ResourceManager.Monitor.Models.ExporterType>
```

Comment:
this name is quite generic, could we add some prefix to specify it?

Reply:
Renamed to `PipelineGroupExporterType`